contrib/DLaaS/pkg/scheduler/util.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/golang/glog"
 
-	"github.com/kubernetes-sigs/kube-batch/contrib/DLaaS/contrib/DLaaS/pkg/scheduler/framework"
+	"github.com/kubernetes-sigs/kube-batch/contrib/DLaaS/pkg/scheduler/framework"
```

Comment:
I cannot successfully build code because of this incorrect path.. did I build it in incorrect way? It should be detected earlier..

Reply:
Looks like developer normally uses 'make kube-batch' to build codes. I tried go build ./... and found this issue. Should be someone's typo.

Reply:
great, good catch!